Sources/MapboxSearch/PublicAPI/SearchOptions.swift

```diff
@@ -68,10 +68,6 @@ public struct SearchOptions {
     /// - Attention: May break engine entity functionality. Do not use without SDK developers agreement
     public var unsafeParameters: [String: String]?
 
-    /// Non-ordered array of attribute sets which describe the level of metadata that will be returned.
-    /// When attributeSets is absent the `.basic` value will be used.
-    public var attributeSets: [AttributeSet]?
```

Comment:
Do I understand correctly, that it was added in #282 `v2.3.0-rc.1` and it is removed in `v2.3.0`, so it's not a breaking change, right?

Reply:
Yes